### DISCUSSION TITLE
When is air 33.1.1.620 supposed to be released as not beta?

### DISCUSSION THREAD

**Discussion by shaymolho** [airsdk/Adobe-Runtime-Support#1216] [Q&A] [answered]
In air sdk [release notes](https://airsdk.harman.com/release_notes) there is no info for the timeline of releasing v620. When will it be the formal latest?

**Comment by 2jfw** [comment, marked as answer]
It actually _is_ the official release.
Also, I don't see the word _beta_ anywhere..

**Reply by shaymolho** [reply]
See the [discussion](https://forum.starling-framework.org/d/22716-new-air-release/89). Apparently there are some bugs so it's not the official recommendation - when you go to the download page it's 575

**Reply by ajwfrost** [reply]
620 was a bit of an interim release, it included some of the changes needed for iOS15 but we couldn't put in the final changes until Apple had issued the final iphoneos SDK and Xcode 13. So, we didn't push that to the 'latest' version that you get to from the normal downloads link. (But it's not really a "beta" or "prerelease" version since those imply you can't create/release apps built with them...)

There's an update, 33.1.1.633, that will hit the public website this afternoon, which contains those final Xcode 13 updates plus a few other minor fixes following the 33.1.1.620 version.

**Reply by 2jfw** [reply]
> See the [discussion](https://forum.starling-framework.org/d/22716-new-air-release/89). Apparently there are some bugs so it's not the official recommendation - when you go to the download page it's 575

I missed that, thanks for point out!

**Reply by ajwfrost** [reply]
FYI ... out now :-)

**Reply by 42retfa** [reply]
> FYI ... out now :-)

Thx Andrew !
